### PR TITLE
Thousand Separators and MIFOSX-1555 - UI On Group

### PR DESCRIPTION
**Pull request by Lajeo**
Variations Thousand separators added

### CODE REVIEW

File: app/views/accounttransfers/view_standinginstruction.html [hunk 1]
```diff
@@ -99,7 +99,7 @@
             </tr>
             <tr>
                 <td>{{ 'label.input.amount' | translate }}</td>
-                <td>{{standinginstruction.amount}}</td>
```
Comment (vishwasbabu) [inline app/views/accounttransfers/view_standinginstruction.html]: small type here

File: app/views/loans/newjlgloanaccount.html [hunk 2]
```diff
@@ -76,12 +78,12 @@
...
-                        <th>Client Name</th>
-                        <th>Loan Purpose</th>
-                        <th>Loan Amount </th>
-                        <th>Charges</th>
+                        <th>{{'label.heading.tickclient' | translate}}</th>
+                        <th>{{'label.heading.clientid' | translate}}</th>
+                        <th>{{'label.heading.clientname' | translate}}</th>
+                        <th>{{'label.heading.loanpurpose' | translate}}</th>
+                        <th>{{'label.heading.loanamount' | translate}}</th>
+                        <th>{{'label.heading.charges' | translate}}</th>
```
Comment (vishwasbabu) [inline app/views/loans/newjlgloanaccount.html]: thanks for catching this!

**Comment by vishwasbabu** [comment]
Thanks for this @Lajeo :+1:, the changes are spot on

Happy to see your first pr to the community app :)

**Comment by Lajeo** [comment]
Thanks @Vishwas, its been a strugle to send the pr but am okay now with it. The problem was i didn't want to read instruction for pr on Github. Am now working on https://mifosforge.jira.com/browse/MIFOSX-1440, please check out the screenshot on the above link on Jira and comment if i should continue and if am doing the right thing.
Thanks

**Pull request merged**